packages/api/src/services/repository-location/repository-location.service.ts

```diff
@@ -1,18 +1,20 @@
 import {RepositoryLocationRestService} from './repository-location-rest.service';
 import {RepositoryLocation} from '../../models/repository-location';
-import {RepositoryLocationMapper} from './repository-location.mapper';
-import {Service} from '../service';
-import {ServiceProvider} from '../../service.provider';
+import {RepositoryLocationMapper} from './mappers/repository-location.mapper';
+import {Service} from '../../providers/service/service';
+import {MapperProvider, ServiceProvider} from '../../providers';
 
 /**
  * The RepositoryLocationService class is responsible for fetching repository-location-related data from the backend
  * and mapping the responses to application models.
  */
 export class RepositoryLocationService implements Service {
   private locationRestService: RepositoryLocationRestService;
+  private repositoryLocationMapper: RepositoryLocationMapper;
 
   constructor() {
     this.locationRestService = ServiceProvider.get(RepositoryLocationRestService);
+    this.repositoryLocationMapper = MapperProvider.get(RepositoryLocationMapper);
```

Comment:
I think we don't need to cache the mappers, but you can leave it this way for now.